addons/udes_stock/models/product_category.py

```diff
@@ -7,3 +7,100 @@ class ProductCategory(models.Model):
     active = fields.Boolean(
         default=True, string="Active", help="Display in list views or searches."
     )
+    u_suggest_locations = fields.Boolean(
+        string="Enable Suggest Locations",
+        default=False,
+        help="Computed suggest locations setting to use for this category.",
```

Comment:
It would be useful to explain what this field does and how it is computed and expose that to the help text